src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -48,6 +48,12 @@
       <PdbFile Condition="'$(PdbFile)' == '' AND '$(OutputType)' == 'winmdobj' AND '$(_DebugSymbolsProduced)' == 'true'">$(IntermediateOutputPath)$(TargetName).compile.pdb</PdbFile>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <LangVersion Condition="'$(LangVersion)' == '' AND
+        (('$(TargetFrameworkIdentifier)' == '.NETCore' AND '$(TargetFrameworkVersion)' == 'v3.0') OR
```

Comment:
This should be .NETCoreApp.

Reply:
It definitely was when I tested it... I screwed something with git here.